contributing.md

```diff
@@ -7,3 +7,18 @@
 
 ### 2 The more involved but proper way
 * [Video](https://www.youtube.com/watch?v=mENDYhfxH-o)
+
+#### Short description of the proper way
```

Comment:
I definitely agree that there should be a written reference offered here, but I think it would be more useful to link to existing resources for this work flow. 

Could you add a link to the written workflow in The Odin Project (https://www.theodinproject.com/courses/ruby-programming/lessons/using-git-in-the-real-world#a-git-workflow-for-open-source-contribution) as well as to this blog post that seems to describe the flow well (https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/)? 

If you find other resources helpful, please feel free to add those here too!

Reply:
Would you like me to delete the summary of the video? I feel like the articles are quite long. The idea was to have a very short summary of the commands themself without a long description, since the person already watched the video once or twice before and just needs to remember the workflow

Reply:
The issue is that this link (https://www.theodinproject.com/courses/ruby-programming/lessons/using-git-in-the-real-world#a-git-workflow-for-open-source-contribution) makes it harder to grasp the concept, because there is at least one step added that is not present in the videos: adding a feature branch.

Let me know how I should proceed. We can discard this change alltogether, or only use the links as you suggested.

Reply:
I have updated contributing.md as per your suggestions :)

Reply:
Creating a feature branch to work in is part of the correct Git workflow, so I do think it's worth including that extra step. 

As far as offering a quick reminder, my hope was that the blog post would provide that: 
![image](https://user-images.githubusercontent.com/42787645/71486339-c8508100-27e3-11ea-9d45-bbc1d4621991.png)

Do you think that's sufficient or would something else be more helpful to you here?

Reply:
Yes, I guess that's totally sufficient! I added the two links. I hope we can get the changes approved. Also merry christmas, leila-alderman!